hera_pspec/utils.py

```diff
@@ -12,6 +12,45 @@
 from datetime import datetime
 
 
+
+def clean_inv_mat(nchan, df, filter_centers, filter_widths, filter_factors):
```

Comment:
lets think of a more apt name for the matrix...

Reply:
I'm going with sync_downweight_mat